Tests/FunctionalTests/Sources/FTRUIWebViewTest.m

```diff
@@ -108,6 +108,13 @@ - (void)DISABLED_testSynchronizationWhenSwitchingBetweenLoadingMethods {
       assertWithMatcher:grey_sufficientlyVisible()];
 }
 
+- (void)testPDFIdlesCorrectly {
```

Comment:
I'd recommend splitting this up with local only and network-only scenarios (for an internal use-case)